frontend/ember-cli-build.js

```diff
@@ -14,6 +14,9 @@ module.exports = function(defaults) {
     },
     babel: {
       optional: ['es7.asyncFunctions', 'es7.decorators']
+    },
+    'ember-uikit': {
+      importUIkitJS: true
```

Comment:
This is the default value.. It's a documentation issue fixed in https://github.com/adfinis-sygroup/ember-uikit/pull/45